### ISSUE TITLE
Implementing composed rules

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1349]
The ACT Task force has asked for me to look into the issue that atomic rules that don't map directly to an SC, but do so through a composite rule have no implementation data. I think I figured out a solution, which I'd like to run by this group.

Similar to how we say inapplicable examples have to satisfy all accessibility requirements of that rule, I think all failed examples of composed rules must fail the composite rules they are part of. For example in the "[Autoplay has a control](https://act-rules.github.io/rules/4c31df)" rule, none of the failed examples should have audio that's shorter than 3 seconds. I think we already do this, I'd be surprise if that wasn't already the case anywhere, but this would make sure we can do the mapping on these.

Second thing to do here is to start including test cases of all the atomic rules in the list of test cases of the composite rule. That'll require an update to the script that creates testcases.json.

**Comment by Jym77** [comment]
That looks strongly tied to having composite rules which are just an "or" of the atomic rules and may become problematic with other kinds of composite rules.

For example, a composite rule could have an expectation:
> Each target that passe rule A also passes rule B

in that case, targets that **fail** rule A actually **pass** the composite rule.

Composite rules are allowed any kind of logical connection between the outcomes of the input rules, so we shouldn't tie ourselves to a big "or" even though this is the only connection we,ve used so far.

**Comment by WilcoFiers** [comment]
Good point, but we don't have such a rule. I think we can just adopt this for now, and cross that bridge when we get there, no?

**Comment by Jym77** [comment]
Sure, we can always try and change if it becomes inconvenient.

**Comment by WilcoFiers** [comment]
TODO: For the migration, an integration can only be considered a consistent implementation if it is consistent with all the atomic rules too.
